src/main/webapp/WEB-INF/app/views/admin/settings/organization/manageOrganization.html

```diff
@@ -68,7 +68,12 @@ <h3 class="panel-title">Manage {{getSelectedOrganization().name}}</h3>
 							    ng-if="getSelectedOrganization().id !== 1"
 								type="button"
 								class="btn btn-danger"
-								ng-click="openModal('#organizationConfirmDeleteModal')">Delete Organization</button>
+                ng-click="openModal('#organizationConfirmDeleteModal')">Delete Organization</button>
```

Comment:
Shouldn't the restore default workflow be under the Manage Workflow tab?

Reply:
Valid concerns, all.  These have been addressed with subsequent commits.